omenai-mobile/screens/register/components/inputForm/individual/IndividualForm.tsx

```diff
@@ -19,31 +31,70 @@ export default function IndividualForm() {
                     onInputChange={setName} 
                     placeHolder='Enter your full name'
                     value={individualRegisterData.name}
+                    handleBlur={() => {
+                        const validationMessage = validateText(individualRegisterData.name)
+                        if(validationMessage.length > 0){
+                            setFormErrors(prev => ({...prev, name: validationMessage[0]}))
+                        }else{
+                            setFormErrors(prev => ({...prev, name: ''}))
+                        }
+                    }}
+                    errorMessage={formErrors.name}
                 />
                 <Input 
                     label='Email address' 
                     keyboardType='email-address' 
                     onInputChange={setEmail} 
                     placeHolder='Enter your email address'
                     value={individualRegisterData.email}
+                    handleBlur={() => {
+                        const validationMessage = validateEmail(individualRegisterData.email)
+                        if(validationMessage.length > 0){
+                            setFormErrors(prev => ({...prev, email: validationMessage[0]}))
+                        }else{
+                            setFormErrors(prev => ({...prev, email: ''}))
+                        }
+                    }}
+                    errorMessage={formErrors.email}
                 />
                 <PasswordInput 
                     label='Password' 
                     onInputChange={setPassword} 
                     placeHolder='Enter password'
                     value={individualRegisterData.password}
+                    handleBlur={() => {
+                        const validationMessage = validatePassword(individualRegisterData.password)
+                        if(validationMessage.length > 0){
+                            setFormErrors(prev => ({...prev, password: validationMessage[0]}))
+                        }else{
+                            setFormErrors(prev => ({...prev, password: ''}))
+                        }
+                    }}
+                    errorMessage={formErrors.password}
                 />
                 <PasswordInput 
                     label='Confirm password' 
                     onInputChange={setConfirmPassword} 
                     placeHolder='Enter password again'
                     value={individualRegisterData.confirmPassword}
+                    handleBlur={() => {
+                        const validationMessage = validateConfirmPassword(individualRegisterData.password, individualRegisterData.confirmPassword)
```

Comment:
In sticking with the DRY (Don't repeat yourself) convention, please convert this into a single function that accepts the necessary parameters and use it across all handleBlur functions. This block of code is used repetitively. The validate function from validatorGroup.ts is designed to receive the label, value and confirm, arguments and then automatically assign the appropriate validator check to it, so you don't have to repetitively invoke the specific validator for each input.

If you require more clarity, don't hesitate to reach out :)